wine/PKGBUILD

```diff
@@ -0,0 +1,157 @@
+            '6e54ece7ec7022b3c9d94ad64bdf1017338da16c618966e8baf398e6f18f80f7b0576edf1d1da47ed77b96d577e4cbb2bb0156b0b11c183a0accf22654b0a2bb'
+            'bdde7ae015d8a98ba55e84b86dc05aca1d4f8de85be7e4bd6187054bfe4ac83b5a20538945b63fb073caab78022141e9545685e4e3698c97ff173cf30859e285'
+            '446b3d4ce7ed7d3cdd395e97ccc773a044cd848a107045e84d02d3f33473a0214d5f64179a301fb953aebe702b92eff0f2a614eca4f968bba064916bb8ca5624')
+validpgpkeys=(5AC1A08B03BD7A313E0A955AF5E6E9EEB9461DD7
+              DA23579A74D4AD9AF9D3F945CEFAC8EAAF17519D)
+
+pkgdesc="A compatibility layer for running Windows programs"
+url="https://www.winehq.org"
+arch=(x86_64)
+options=(staticlibs !lto)
+license=(LGPL-2.1-or-later)
+depends=(
+  desktop-file-utils
+  fontconfig      #lib32-fontconfig
+  freetype2       #lib32-freetype2
+  gcc-libs        #lib32-gcc-libs
+  gettext         #lib32-gettext
+  libpcap         #lib32-libpcap
+  libunwind       #lib32-libunwind
+  libxcursor      #lib32-libxcursor
+  libxkbcommon    #lib32-libxkbcommon
+  libxi           #lib32-libxi
+  libxrandr       #lib32-libxrandr
+  wayland         #lib32-wayland
+)
+makedepends=(autoconf bison perl flex mingw-w64-gcc
+  git
+  alsa-lib              #lib32-alsa-lib
+  ffmpeg
+  gnutls                #lib32-gnutls
+  gst-plugins-base-libs #lib32-gst-plugins-base-libs
+  libcups               #lib32-libcups
+  libgphoto2
+  libpulse              #lib32-libpulse
+  libxcomposite         #lib32-libxcomposite
+  libxinerama           #lib32-libxinerama
+  libxxf86vm            #lib32-libxxf86vm
+  mesa                  #lib32-mesa
+  mesa-libgl            #lib32-mesa-libgl
+  opencl-headers
+  opencl-icd-loader     #lib32-opencl-icd-loader
+  pcsclite              #lib32-pcsclite
+  python
+  samba
+  sane
+  sdl2                  #lib32-sdl2
+  unixodbc
+  v4l-utils             #lib32-v4l-utils
+  vulkan-headers
+  vulkan-icd-loader     #lib32-vulkan-icd-loader
+)
+makedepends+=(
+  ntsync-header
+)
+optdepends=(
+  alsa-lib              #lib32-alsa-lib
+  alsa-plugins          #lib32-alsa-plugins
+  cups                  #lib32-libcups
+  dosbox
+  ffmpeg
+  gnutls                #lib32-gnutls
+  gst-plugins-bad
+  gst-plugins-base      #lib32-gst-plugins-base
+  gst-plugins-base-libs #lib32-gst-plugins-base-libs
+  gst-plugins-good      #lib32-gst-plugins-good
+  gst-plugins-ugly
+  libgphoto2
+  libpulse              #lib32-libpulse
+  libxcomposite         #lib32-libxcomposite
+  libxinerama           #lib32-libxinerama
+  opencl-icd-loader     #lib32-opencl-icd-loader
+  pcsclite              #lib32-pcsclite
+  samba
+  sane
+  sdl2                  #lib32-sdl2
+  unixodbc
+  v4l-utils             #lib32-v4l-utils
+  wine-gecko
+  wine-mono
+)
+optdepends+=(
+  NTSYNC-MODULE
+)
+makedepends=(${makedepends[@]} ${depends[@]})
+install=wine.install
+
+prepare() {
+  # Get rid of old build dirs
+  rm -rf $pkgname-{32,64}-build
```

Comment:
Splitting into 32/64 bit is not needed here anymore, I guess.

Reply:
Indeed